scripts/smoke_test.py

```
@@ -0,0 +1,189 @@
+
+        # 各主要ページのロードとコンテンツ確認
+        for name, url, check_selector in PAGES:
+            print(f"→ {name}: {url}")
+            page.goto(url, wait_until="networkidle", timeout=20000)
+            # CSS セレクタの場合は動的描画の完了を明示的に待つ
+            # （networkidle 直後では認証判定・遅延 import が未完了な場合がある）
+            if check_selector.startswith("["):
+                try:
+                    page.wait_for_selector(check_selector, state="visible", timeout=10000)
+                except Exception:
+                    pass  # タイムアウト時は count() で false を検出して失敗として記録
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)